src/views/OrderDetail.vue

```diff
@@ -4,58 +4,40 @@
       <ion-toolbar>
         <ion-back-button default-href="/" slot="start" />
         <ion-title>{{ $t("Order details") }}</ion-title>
+        <ion-buttons slot="end">
+          <ion-button @click="openOrderItemRejHistoryModal()">
+            <ion-icon slot="icon-only" :icon="timeOutline" />
+          </ion-button>
+          <ion-button v-if="!isDesktop" :disabled="!hasPermission(Actions.APP_ORDER_UPDATE)" @click="updateOrder(order)">
+            <ion-icon slot="icon-only" color="danger" :icon="bagRemoveOutline" />
+          </ion-button>
+        </ion-buttons>
       </ion-toolbar>
     </ion-header>
     <ion-content>
-      <main>
-        <ion-list>
-          <ion-item lines="none">
-            <ion-label class="ion-text-wrap">
-              <h2>{{ order.customer?.name }}</h2>
-              <p>{{ order.orderName ? order.orderName : order.orderId }}</p>
-            </ion-label>
-            <ion-badge v-if="order.placedDate" color="dark" slot="end">{{ timeFromNow(order.placedDate) }}</ion-badge>
-          </ion-item>
-        </ion-list>
-        <ion-item v-if="order.shippingInstructions" color="light" lines="none">
-          <ion-label class="ion-text-wrap">
-            <p class="overline">{{ $t("Handling Instructions") }}</p>
-            <p>{{ order.shippingInstructions }}</p>
-          </ion-label>
-        </ion-item>
-        <ion-item v-if="customerEmail" lines="none">
-          <ion-icon :icon="mailOutline" slot="start" />
-          <ion-label>{{ customerEmail }}</ion-label>
-          <ion-button
-            fill="outline"
-            slot="end"
-            color="medium"
-            @click="copyToClipboard(customerEmail)"
-          >
-            {{ $t("Copy") }}
-          </ion-button>
-        </ion-item>
+      <main :class="{ 'desktop-only' : isDesktop }">
+        <OrderInfo v-if="!isDesktop" :orderId="$route.params.orderId" />
```

Comment:
Is there a way to do this without using order info component twice?

Reply:
Sir, previously I have tried this approach - https://github.com/hotwax/bopis/pull/302/commits/428740b95d81d37c91fa974119d7377084054a9b

video of UI - 
[nothing.webm](https://github.com/hotwax/bopis/assets/67117461/425fd91e-aefd-4ebc-9164-7288c74335ec)